ogre/include/ignition/rendering/ogre/OgreScene.hh

```diff
@@ -142,6 +142,10 @@ namespace ignition
       protected: virtual MarkerPtr CreateMarkerImpl(unsigned int _id,
                      const std::string &_name);
 
+      // Documentation inherited
+      protected: virtual LidarVisualPtr CreateLidarVisualImpl(unsigned int _id,
+                     const std::string &_name) override;
```

Comment:
the `override` keyword is correct and the right thing to do. It's however producing warnings on [homebrew build](https://build.osrfoundation.org/job/ignition_rendering-ci-pr_any-homebrew-amd64/735/warnings10Result/new/). To resolve the warnings, we need to add the keyword to other functions, which would end up increasing the size of this PR. Let's remove `override` in this PR so we can get a clean homebrew build, and we can create a separate PR to fix these warnings later.